OutOfSchool/OutOfSchool.Common/PermissionModule/PermissionManagement/Permissions.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace OutOfSchool.Common.PermissionsModule
+{
+    public enum Permissions : short
```

Comment:
Just a question for discussion: separate groups for permissions (have separate enums for each group). That way enums gonna be smaller and we can use flag enums for fast bitwise permissions checks within a group.